pipelines/main/misc/analyzegc.yml

```
@@ -19,7 +19,7 @@ steps:
                 - "/cache/repos:/cache/repos"
         commands: |
           echo "--- Install in-tree LLVM dependencies"
-          make --output-sync -j$${JULIA_CPU_THREADS:?} -C deps install-llvm install-clang install-llvm-tools install-libuv install-utf8proc install-unwind
+          make --output-sync -j$${JULIA_CPU_THREADS:?} -C src install-analysis-deps
           echo "+++ run clangsa/analyzegc"
           make --output-sync -j$${JULIA_CPU_THREADS:?} -C test/clangsa
           make --output-sync -j$${JULIA_CPU_THREADS:?} -C src analyzegc
```

could you also make this use the more general name? (add in v1.9 by JuliaLang/julia#44900)
```suggestion
          make --output-sync -j$${JULIA_CPU_THREADS:?} -C src analyze
```

---

👍